saleor/order/utils.py

```diff
@@ -147,16 +148,24 @@ def recalculate_order_prices(order: Order, **kwargs):
             assigned_order_discount.save(update_fields=["value", "amount_value"])
 
 
-def recalculate_order(order: Order, **kwargs):
+def recalculate_order(order: Order, invalidate_prices: bool = False, **kwargs):
     """Recalculate and assign total price of order.
 
     Total price is a sum of items in order and order shipping price minus
     discount amount.
 
     Voucher discount amount is recalculated by default. To avoid this, pass
     update_voucher_discount argument set to False.
+
+    If you want to invalidate order prices, pass
+    invalidate_prices_data argument set to True.
     """
 
+    invalidate_updated_fields = []
+
+    if invalidate_prices is not None:
```

Comment:
I think this condition should depend on the `invalidate_prices` flag's value, no? Now we will enter this condition for both `False` and `True` values of the argument (in both cases it won't be a `None`).